### PR TITLE
[tests-only][full-ci] adding test to remove old versions of files from a space using space-id using CLI 

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR adds test for removing versions of files from a specific space using space-id using CLI command

## Related Issue

- Coverage for : https://github.com/owncloud/ocis/issues/9472
- Part of : https://github.com/owncloud/ocis/issues/9529

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/CliContext.php [hunk 1]
```diff
@@ -139,6 +139,24 @@ public function theAdministratorRemovesTheVersionsOfFileUsingFileId($file, $user
 		$this->featureContext->setResponse(CliHelper::runCommand($body));
 	}
 
+	/**
+	 * @When the administrator removes the versions of the files of user :user in space :space using space-id using CLI
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/CliContext.php:143]:
```suggestion
	 * @When the administrator removes the file versions of space :space using the CLI
```

File: tests/acceptance/features/bootstrap/CliContext.php [hunk 2]
```diff
@@ -139,6 +139,24 @@ public function theAdministratorRemovesTheVersionsOfFileUsingFileId($file, $user
...
+	 *
+	 * @param string $user
+	 * @param string $space
+	 *
+	 * @return void
+	 */
+	public function theAdministratorRemovesTheVersionsOfFilesInSpaceUsingSpaceId($user, $space) {
+		$path = $this->featureContext->getStorageUsersRoot();
+		$spaceId = $this->spacesContext->getSpaceIdByName($user, $space);
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/CliContext.php:152]:
```suggestion
		$spaceId = $this->spacesContext->getSpaceIdByName(<admin-user>, $space);
```

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/bootstrap/CliContext.php [hunk 3]
```diff
@@ -139,6 +139,24 @@ public function theAdministratorRemovesTheVersionsOfFileUsingFileId($file, $user
 		$this->featureContext->setResponse(CliHelper::runCommand($body));
 	}
 
+	/**
+	 * @When the administrator removes the file versions of space :space using space-id using the CLI
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/CliContext.php:143]:
```suggestion
	 * @When the administrator removes the file versions of space :space using the CLI
```

**Comment by prashant-gurung899** [comment]
@saw-jan In the test,instead of uploading only one file in the space and delete the versions of it, I have added steps to upload another file in the space so that we can know if it will delete all the versions of files from that space or not.

File: tests/acceptance/features/cliCommands/removeFileVersions.feature [hunk 4]
```diff
@@ -32,3 +33,32 @@ Feature: remove file versions via CLI command
...
+
+
+  Scenario: remove all versions of files from space using space-id
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And using spaces DAV path
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has uploaded file with content "This is version 1" to "file.txt"
+    And user "Alice" has uploaded file with content "This is version 2" to "file.txt"
+    And user "Alice" has uploaded file with content "This is version 3" to "file.txt"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 1" to "anotherFile.txt"
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeFileVersions.feature:45]:
```suggestion
    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 1" to "lorem.txt"
```

File: tests/acceptance/features/cliCommands/removeFileVersions.feature [hunk 5]
```diff
@@ -32,3 +33,32 @@ Feature: remove file versions via CLI command
...
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 2" to "anotherFile.txt"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 3" to "anotherFile.txt"
+    And we save it into "FILEID"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 1" to "anotherFile1.txt"
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeFileVersions.feature:49]:
```suggestion
    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 1" to "epsum.txt"
```

**Review by saw-jan — CHANGES_REQUESTED** [review]
let use distinctive names for different files instead of appending numbers.

File: tests/acceptance/features/cliCommands/removeFileVersions.feature (hunk 4 above)
Comment (prashant-gurung899) [reply tests/acceptance/features/cliCommands/removeFileVersions.feature:45]: Sure. Thanks for the suggestion.

File: tests/acceptance/features/cliCommands/removeFileVersions.feature [hunk 6]
```diff
@@ -1,10 +1,12 @@
-@env-config
+#@env-config
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeFileVersions.feature:1]:
```suggestion
@env-config
```

File: tests/acceptance/features/cliCommands/removeFileVersions.feature [hunk 7]
```diff
@@ -32,3 +33,32 @@ Feature: remove file versions via CLI command
     When user "Alice" gets the number of versions of file "anotherFile.txt"
     Then the HTTP status code should be "207"
     And the number of versions should be "2"
+
+
+  Scenario: remove all versions of files from space using space-id
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeFileVersions.feature:38]:
```suggestion
  Scenario: remove all versions of files from a space
```

File: tests/acceptance/features/cliCommands/removeFileVersions.feature (hunk 6 above)
Comment (prashant-gurung899) [reply tests/acceptance/features/cliCommands/removeFileVersions.feature:1]: HAHHA. forgot about this. i was trying something else by commenting it.

File: tests/acceptance/features/cliCommands/removeFileVersions.feature [hunk 8]
```diff
@@ -32,3 +33,32 @@ Feature: remove file versions via CLI command
...
+    And user "Alice" has uploaded file with content "This is version 2" to "file.txt"
+    And user "Alice" has uploaded file with content "This is version 3" to "file.txt"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 1" to "lorem.txt"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 2" to "lorem.txt"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 3" to "lorem.txt"
+    And we save it into "FILEID"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 1" to "epsum.txt"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 2" to "epsum.txt"
+    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 3" to "epsum.txt"
+    And we save it into "FILEID1"
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeFileVersions.feature:52]: can we rename fileid variable as well? `LOREM_FILEID` and `EPSUM_FILEID`
```suggestion
    And we save it into "LOREM_FILEID"
    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 1" to "epsum.txt"
    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 2" to "epsum.txt"
    And user "Alice" has uploaded a file inside space "projectSpace" with content "This is version 3" to "epsum.txt"
    And we save it into "EPSUM_FILEID"
```

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**